packages/UI/src/components/organisms/PositionCard/index.ts

```diff
@@ -0,0 +1,10 @@
+export { default } from './PositionCard';
+
+export interface PositionCardProps {
+  // interaction: boolean;
+  // status: boolean;
+  interaction: 'all-h' | 'all-v' | 'save-only' | 'none';
+  style: 'default' | 'minimal' | 'mobile' | 'long';
```

Comment:
@Nouri-Anouar 
This should be optional right? if nothing is passed then default styles should be used

Reply:
I think we should simplify this to be just "default" | "minimal", default will implicitly have "long" and  "mobile" handled with media queries.
What do you think?

Reply:
@Nouri-Anouar 
firstly `style` is a reserved word in react compoents
and since we only have minimal we can have it as boolean
like this
{
minimal?: boolean
}